windows-iotcore/tutorials/quickstarter/DeviceSetup.md

```diff
@@ -68,10 +68,10 @@ Use the [Windows Device Portal](../../manage-your-device/DevicePortal.md) to con
 
 1. Download the Windows 10 IoT Core Dashboard [here](https://developer.microsoft.com/en-us/windows/iot/Downloads).
 2. Once downloaded, open the Dashboard and select "Qualcomm DragonBoard 410c". Then _sign in as a Windows Insider_. You need to be signed in as an insider in order to flash DragonBoard 410c. 
-3. Connect the Qualcomm board to the developer machine using a microUSB cable. Power on the DragonBoard to a 12V (> 1A) power supply.
-4. Turn the DragonBoard to flash mode by pressing the volume up (+) button while powering on the device. The device - when connected to a display - will show the image of a hammer, a lightning bolt, and a cog.
+3. Connect the Qualcomm board to the developer machine using a 5V microUSB cable. Power on the DragonBoard to a 12V (> 1A) power supply.
```

Comment:
This should be the following:
3. Connect the Qualcomm board to the developer machine using a microUSB cable.
4. Power on your Dragonboard using a 12V (>1A) power supply while holding down the volume up (+).  The device - when connected to a display - should show the image of a hammer, a lightning bolt, and a cog.